src/main/java/org/kohsuke/github/GHPullRequestReviewCommentReactions.java

```diff
@@ -0,0 +1,120 @@
+package org.kohsuke.github;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.net.URL;
+
+/**
+ * Reactions for a Pull Request Review comment.
+ *
+ * @author Vasilis Gakias
+ * @see <a href="https://docs.github.com/en/rest/pulls/comments#get-a-review-comment-for-a-pull-request">API
+ *      documentation</a>
+ * @see GHPullRequestReviewComment
+ */
+public class GHPullRequestReviewCommentReactions {
```

Comment:
I am not seeing the `reactions` field listed in the doc link mentioned here.  
This page talks about reactions API: 
https://docs.github.com/en/rest/reactions#list-reactions-for-a-pull-request-review-comment